lstchain/reco/utils.py

```diff
@@ -271,13 +224,9 @@ def camera_to_sky(pos_x, pos_y, focal, pointing_alt, pointing_az):
     y = np.array([1,1]) * u.m
 
     """
-    pointing_direction = SkyCoord(alt=pointing_alt, az=pointing_az,
-                               frame=horizon_frame
-    )
+    pointing_direction = SkyCoord(alt=pointing_alt, az=pointing_az, frame=horizon_frame)
 
-    camera_frame = CameraFrame(focal_length=focal,
-                               telescope_pointing=pointing_direction
-    )
+    camera_frame = CameraFrame(focal_length=focal, telescope_pointing=pointing_direction)
```

Comment:
same as above

Reply:
(also a couple below)

Reply:
Yes, the parenthesis shouldn't hang there.
Small things, I just spotted while refactoring and my IDE complained so I fixed it...